test/intl402/Segmenter/prototype/segment/segment-word-iterable.js

```diff
@@ -29,22 +29,12 @@ for (const text of [
     "법원 “다스 지분 처분권·수익권 모두 MB가 보유”", // Korean
     ]) {
   let segments = [];
-  // Create another %SegmentIterator% to compare with result from the one that
-  // created in the for of loop.
-  let iter = seg.segment(text);
-  let prev = 0;
   for (const v of seg.segment(text)) {
-    assert(["word", "none"].includes(v.breakType), v.breakType);
+    assert.sameValue("boolean", typeof v.isWordLike);
+    assert([true, false].includes(v.isWordLike), v.isWordLike);
```

Comment:
If this assert ever executes, `[true, false].includes(v.isWordLike)` will always be `true`, given we asserted above that `typeof v.isWordLike` is `boolean`. I think it is redundant if we assert it here as well. Also, I'm a bit confused why second argument is `v.isWordLike`. IIRC, the signature for this function is `function assert(mustBeTrue, message)` and passing a boolean to `message` seems a bit off.

Reply:
> If this assert ever executes, `[true, false].includes(v.isWordLike)` will always be `true`, given we asserted above that `typeof v.isWordLike` is `boolean`. I think it is redundant if we assert it here as well. Also, I'm a bit confused why second argument is `v.isWordLike`. IIRC, the signature for this function is `function assert(mustBeTrue, message)` and passing a boolean to `message` seems a bit off.

Yes, it is redundant so I will remove it. 
Yes, the second argument is the message, I intend to print out the value in the error but since as you point out it is redundant and I will just remove it now it is not not important.